database/index.js

```diff
@@ -1,104 +1,110 @@
-const mongoose = require('mongoose');
-var { dbUrl, user, pass } = require('./config.js')
+var configObject = require('./config.js')
+const { Client } = require('pg');
+const client = new Client(configObject);
```

Comment:
One suggestion for modularity, you can separate your save, read, update, delete into a different file, and import them here, to allow easy switching between dbs, if that sounds interesting to you.